hvpy/utils.py

```diff
@@ -127,3 +129,25 @@ def create_events(events: List[Union[EventType, str, tuple]]) -> str:
             raise ValueError(f"{event} is not a EventType or str or two-length tuple")
     # Strips the final comma
     return constructed_events[:-1]
+
+
+def save_file(data: bytearray, filename: str, overwrite: Optional[bool] = None) -> None:
```

Comment:
We should also check that filename works with a Path object

Reply:
We accept `filename` as an `str` and string works with Path, how can we check it?

Reply:
You pass in a Path object in a test and will need to update the type hint as well.